src/directives/badge/badge.directive.ts

```diff
@@ -192,6 +192,8 @@ export class BadgeDirective implements AfterViewInit, OnChanges, OnDestroy {
         } else {
             this._renderer.removeStyle(this._badgeElement, 'border-color');
         }
+
+        this._renderer.setStyle(this._badgeElement, 'background-clip', this._badgeBorderColor ? 'padding-box' : 'border-box');
```

Comment:
Can we use `padding-box`, doesn't look well supported?

![image](https://user-images.githubusercontent.com/20795331/80620917-83ad4e80-8a3e-11ea-9b79-e7c2b3a28cc3.png)

Reply:
Nevermind... I see this is background-clip, not box-sizing

Reply:
https://www.w3schools.com/CSSref/css3_pr_background-clip.asp

I think this is ok for back to IE9 and early versions of other browsers.

Reply:
Yeah, I initially I thought it was box sizing being set, so just ignore this comment